src/functions-and-arrays.js

```diff
@@ -1,38 +1,127 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+
+
+function maxOfTwoNumbers(harrySize, bobSize) {
+  if (harrySize < bobSize) {
+    return bobSize;
+  } 
+  
+  else if (bobSize < harrySize) {
+    return harrySize;
+   }
+
+   else  {
+     return harrySize;
+   }
+  
+};
+
+let harrySize = 7;
+let bobSize = 5;
+
+console.log(maxOfTwoNumbers(harrySize, bobSize));
+
 
 
 
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+//const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+
+/*findLongestWord("mystery brother aviator crocodile pearl orchard crackpot"); */
+
+
+const str = "mystery brother aviator crocodile pearl orchard crackpot";
+
+const myArray = str.split(" ");
+
+
+
+
+
+
+
+
+
+
+function longestStringForLoop(myArray) {
+  let longestWord = " ";
+ 
+  for(var i = 0; i < myArray.length; i++){
+    if(myArray[i].length > longestWord){
+	longestWord = myArray[i].length;
+     }
+  }
+  return longestWord;
+}
+
+console.log(longestStringForLoop(myArray));
+
 
-function findLongestWord() {}
 
 
 
 // Iteration #3: Calculate the sum
+/*function sum() {}*/
+
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+let sum = 0;
+
+  for (let i = 0; i < numbers.length; i++) {
+    sum += numbers[i];
+  }
+  
+
+console.log(sum);
```

Comment:
In iteration 3, your function disappeared... you have some logic to calculate sum , but no function itself..

And once you have your logic inside the function, do not forget to `return` the result ;)